### ISSUE TITLE
human-panic swallows helpful error message

### ISSUE THREAD

**Issue by danreeves** [yoshuawuyts/crossgen#16]
# Bug Report
## Your Environment
| Software         | Version(s) |
| ---------------- | ---------- |
| crossgen      | 0.5.0
| Rustc            | rustc 1.30.0-nightly (20dc0c507 2018-09-19)
| Operating System | macOS

## Expected Behavior
When you're missing the `repository` field from `Cargo.toml` you should receive the error `No repository found in Cargo.toml`.

## Current Behavior
You get `human-panic` output pointing to an error report, nether of which contain any reference to the original error message which comes from here:
https://github.com/yoshuawuyts/crossgen/blob/48fa6fca63f5662807d7c0ae8995fa8099bda3c5/src/main.rs#L34

**Comment by yoshuawuyts** [comment]
@danreeves yeah agree this ought to be fixed. Issue about it is open here https://github.com/rust-clique/human-panic/issues/40. A PR to fix this on `human-panic` would be great!

cc/ @spacekookie ping, figure you might be interested in this too.

**Comment by spacekookie** [comment]
@yoshuawuyts can you please configure your bot to be less pessimistic? :joy:

**Comment by yoshuawuyts** [comment]
@spacekookie lol, sure.

We should probably update this file to also include `feature` and `bug` reports. https://github.com/yoshuawuyts/crossgen/blob/master/.github/stale.yml#L3-L7.

As an aside: github templates now also include a way to auto-label issues created with a certain template, so that should hopefully allow us to smoothen out the pipeline.
